.github/workflows/documentation_dev.yml

```diff
@@ -0,0 +1,30 @@
+name: Generate dev documentation
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  api-docs:
```

Comment:
Is there a reason this job is called api-docs when it pushes all docs (not only the API specific ones)?

Reply:
I was rushing a bit to get the documentation working. I think I just copy and pasted from the previous version where I had multiple pipelines for the different parts of the documentation. **Will fix**.